src/index.js

```diff
@@ -118,6 +119,17 @@ async function writeBabelRC(configPath, options) {
   }
 }
 
+async function writeBabelRCJS(configPath) {
+  try {
+    const rawFile = (await pify(fs.readFile)(configPath)).toString('utf8');
```

Comment:
Is there any advantage to doing `fs.readFile` over `fs.readFileSync` ?

Reply:
The only advantage i can see is that it lets the event loop continue during the i/o rather than blocking it.